### DISCUSSION TITLE
Is new release ?

### DISCUSSION THREAD

**Discussion by matzel82** [cedricp/ddt4all#845] [Q&A] [answered]
Is there a new release or database ?

**Comment by KarelSvo** [comment]
You can create [a new release](https://github.com/cedricp/ddt4all/releases) yourself with Inno Setup. The last database is from october '22.

https://jrsoftware.org/isinfo.php and https://github.com/cedricp/ddt4all/blob/master/wininstaller.iss

**Reply by matzel82** [reply]
Where can I load the latest database ?

I can't create a new release... I'm not a developer 😂🙈only a Renault Super-User ❤️

**Reply by Furtif** [reply]
> You can create [a new release](https://github.com/cedricp/ddt4all/releases) yourself with Inno Setup. The last database is from october '22.
> 
> https://jrsoftware.org/isinfo.php and https://github.com/cedricp/ddt4all/blob/master/wininstaller.iss

yup i have 2022-10-25

invited cedric in https://github.com/cedricp/ddt4all/discussions/832#discussioncomment-6139380

**Reply by matzel82** [reply]
> > You can create [a new release](https://github.com/cedricp/ddt4all/releases) yourself with Inno Setup. The last database is from october '22.
> 
> > 
> 
> > https://jrsoftware.org/isinfo.php and https://github.com/cedricp/ddt4all/blob/master/wininstaller.iss
> 
> 
> 
> yup i have 2022-10-25
> 
> 
> 
> invited cedric in https://github.com/cedricp/ddt4all/discussions/832#discussioncomment-6139380
> 
> 

I see only a empty 404 page

**Reply by Furtif** [reply]
> > > You can create [a new release](https://github.com/cedricp/ddt4all/releases) yourself with Inno Setup. The last database is from october '22.
> > 
> > 
> > > 
> > 
> > 
> > > https://jrsoftware.org/isinfo.php and https://github.com/cedricp/ddt4all/blob/master/wininstaller.iss
> > 
> > 
> > yup i have 2022-10-25
> > invited cedric in [#832 (comment)](https://github.com/cedricp/ddt4all/discussions/832#discussioncomment-6139380)
> 
> I see only a empty 404 page

Yep is a private repos only users invited in it.
![78B5F1B0-95B1-452E-A4B9-827CB0F7A73F](https://github.com/cedricp/ddt4all/assets/11375035/9a1989e9-531e-4c10-9a0a-be80be6af848)

**Reply by matzel82** [reply]
Can you send my a invitation?

**Reply by Furtif** [reply]
> Can you send my a invitation?

You have ddt2000 real release? Or only dtt4all?

if ddt2000 release not installed fake server intranet is obsoleted for you but repo data is ok for dtt4all.

**Reply by matzel82** [reply]
I use ddt4all...

I look for a new release or database.

**Reply by Furtif** [reply]
> I use ddt4all...
> 
> I look for a new release or database.

[DDT2000Data](https://github.com/Furtif/DDT2000data/invitations)

**Reply by Furtif** [reply]
> I use ddt4all...
> 
> I look for a new release or database.

Can’t not share normally data base is not included in this code

**Reply by matzel82** [reply]
Is there explaind what I have to do ?

**Reply by Furtif** [reply]
> Is there explaind what I have to do ?

Clone or download repo content and put in dtt4all is in wiki’s pages

**Reply by Furtif** [reply]
Maybe better a ecu.zip for new boys

**Reply by Furtif** [reply, marked as answer]
database as ecu.zip https://drive.google.com/file/d/14AQTXRv4Y0mVnWNeZ6seznKPRd-7ZaKm/view?usp=sharing

copy zip file in to root ddt4all folder.

**Reply by matzel82** [reply]
Is it sufficient to install the latest version, or do I still have to copy the folder with the database into the directory?

Thanks for the work

**Reply by Furtif** [reply]
Yep paste zip in c:/xxxxx/ddt4all if you as installed windows install last is program files /ddt4all

**Reply by matzel82** [reply]
I have copied the ecu.zip to program folder. The date at the top of DDT4All show me the date from 2018. it's okay ?

I can't install the latest release from ddt4 at this machine. It's win7 32 bit (it's to old)

**Reply by matzel82** [reply]
> > I use ddt4all...
> 
> > 
> 
> > I look for a new release or database.
> 
> 
> 
> [DDT2000Data](https://github.com/Furtif/DDT2000data/invitations)

It's that for ddt4 ? I have the folder at this machine. On a other machine I haven't this folder. I have copied the file to my harddrive and overwrite.

**Reply by Furtif** [reply]
Copy zip in old release if is python 32

**Reply by matzel82** [reply]
Also copy ecu.zip to folder AND install python 3.2 together?

Did I understand correctly that's important to install python 32 ?

**Reply by Furtif** [reply]
Not 3.2 but version 32 bits python 3.9 32 bits for example or 3.10 or 11 but not have look if exists 32 bits versions for last python, 3.9 exists.

**Reply by Furtif** [reply]
Maybe need make a win install as 32bits also … but now I m in works

**Reply by Furtif** [reply]
weel delete all go send you a 32 bits win install + db (full) is a python 3.9 as last code in repo.

**Reply by Furtif** [reply]
test this https://drive.google.com/file/d/1fRhj4iZqaGREuQZFEejUd9NBQOmf-YXu/view?usp=drive_link 

not need copy databases is allready in but intallation says for put one click ok and ignore

**Reply by Furtif** [reply]
> I have copied the ecu.zip to program folder. The date at the top of DDT4All show me the date from 2018. it's okay ?
> 
> I can't install the latest release from ddt4 at this machine. It's win7 32 bit (it's to old)
it should be a bit trendy... almost everything is x64 now i don't think 32 bits has a future, but good for an auto garage an old portable pc server perfectly for obd.

I just don't know if all future updates will be good... even python 2.7 hardly installs anymore...

https://github.com/cedricp/ddt4all/discussions/845#discussioncomment-6154185
test it I changed the installer to inno 5 to be compatible with win7 since inno 6 only works with win 10 and higher.

if it's ok I should delete my files in google drive, I don't like to have a lot of delicate things in public.

next release you have to think about making both platforms windows 32 and 64.

**Reply by matzel82** [reply]
> test this https://drive.google.com/file/d/1fRhj4iZqaGREuQZFEejUd9NBQOmf-YXu/view?usp=drive_link 
> 
> 
> 
> not need copy databases is allready in but intallation says for put one click ok and ignore 

Thanks a lot. I have load it.

I get a message at the start of the ddt4 that's missing the api-ms-win-core-path-l1-1-0.dll

**Reply by Furtif** [reply]
> api-ms-win-core-path-l1-1-0.dll

https://answers.microsoft.com/en-us/windows/forum/all/api-ms-win-core-path-l1-1-1ddl-is-missing/b7afd65f-b4f6-4868-8433-587a543532e5

**Reply by Furtif** [reply]
https://www.youtube.com/watch?v=TQ2XAmGDLmU

**Reply by Furtif** [reply]
visibly you have never used visual studio or derivatives in programming, then you need some dependencies to be able to use python 3.9 or higher, watch the video and get the 32-bit version since it uses win7 32-bit cpu arch ... depending on the users that is required or not, for example I don't need it because I installed visual studio on win7 (my old pc) but basically if they are not developers it doesn't matter at all, just decorating the pc and taking up space....
But there are many apps that require vc_redist and not just NetStandard, Framework etc... if you have a win7 pc not updated, that is with the latest microsoft updates, and since microsoft stopped giving support you can use it:
https://legacyupdate.net/

but install dep of python 3.9 32 bits normally issue fixed.

have fun!

**Reply by Furtif** [reply]
in theory this https://github.com/cedricp/ddt4all/releases/tag/V1.02 is a install python 2.x and old code 2.x python (this code not works more in py3.x, becuse print "xxx" changes to print("xx") etc etc etc etc .....)
this https://github.com/cedricp/ddt4all/releases/tag/1.5 is a mutant py 2.x ->3.x (init versions as py 3.x)

**Reply by Furtif** [reply]
legacy update is a open source alternative microsoft at https://github.com/kirb/LegacyUpdate

this update xp, 2000, vista, 7, 8 windows version similar as old windows update stuff

**Reply by Furtif** [reply]
theoretically if a system has all the updates that py 3.x should or should work since py 3.x came out before microsoft for support... but ready... there's everything there if you can't make it work is because you really don't understand anything lol (no offense)

**Reply by Furtif** [reply]
correct me if i'm wrong @cedricp about tags or releases.

**Reply by Furtif** [reply]
if you have your garages in france in the south of france (millau) i want a free inspection.... otherwise i won't be able to help you anymore :) xd!!!

**Reply by Furtif** [reply]
a little more.... he'll ask me to do the diag hehehehe, really, there's everything there if you can't!!! I don't know! 👍

**Reply by matzel82** [reply]
No, I came from Germany. I will try it to fix it. I take a look to the video. I have load the version.... really thanks a lot. I will try it to run at the pc. Otherwise I have to use a other pc.

**Reply by Furtif** [reply]
when you are sure that all the answers are answered, please close the thread, not for nothing but I think there will be many... (given the general cleaning that was given)

Thanks :) 

for any other code related issue, windows, linux, macOS can talk or open another discussion.

As far as various hacks with the app are concerned, I don't know... I just used 0.000001% of ddt4all in real (I changed things from the gas needle, put time + temp in control panel, auto loock doors, and little else or less) but there are people out there who even go to the injectors to consume less gasoline... but don't ask me that.

**Reply by Furtif** [reply]
> No, I came from Germany. I will try it to fix it. I take a look to the video. I have load the version.... really thanks a lot. I will try it to run at the pc. Otherwise I have to use a other pc.

normally this is ok... I have a win7 32 bits pc and it runs all this... the difference is that my win 7 pc was my old pc dev it has all the dev tools + all the latest Microsoft updates.

when I talk about dev tools... I'm talking about everything that goes with csharp, py, go, proto, js, html5, php etc... a dev is never half done... although I'm not very good at python but when if you know some languages, almost everything looks sharp, almost like java, js, almost like python... well, the "almost" is missing, but that's a detail.

**Reply by Furtif** [reply]
otherwise you can go with your pc win 7 in legacy update (of course if you go with win 10 or + it doesn't matter)

in windows 7 open internet explorer go to http://legacyupdate.net/ install what you propose...
will have all the updates until the last one given by microsoft.

**Reply by Furtif** [reply]
the problem with a win xp, vista, 7, 8 system is the https or root certificates.. hence the site is http: and not https: just to be compatible.

![image](https://github.com/cedricp/ddt4all/assets/11375035/5b348830-518b-4d7f-9706-9c66d54b3951)

then I don't know .... how do you come on github ??? this is https, i assume its not with your win 7

the party that has used legacyupdate otherwise does not have access to 95% of the sites on (443) https

**Reply by Furtif** [reply]
this would almost make a good tutorial for using a dedicated old pc 🔢

**Reply by Furtif** [reply]
for other side this https://github.com/cedricp/ddt4all/discussions/845#discussioncomment-6144566

need a list vehicules ok or https://github.com/cedricp/ddt4all/blob/master/main.py#L95 

list actual in code is according "last databases" or :
```chsarp
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;

namespace DDT4ALLGenCars
{
    internal class Program
    {
        static void Main(string[] args)
        {
            /*
            Guid guid = new Guid(new byte[]
            {
                167,
                188,
                92,
                9,
                186,
                87,
                87,
                68,
                155,
                76,
                187,
                165,
                192,
                126,
                65,
                192
            });

            Console.WriteLine(guid.ToString());
            Console.ReadKey();
            var nn = "9d7debbc-c85d-11d1-9eb4-006008c3a19a";

            byte[] keyBytes = Encoding.ASCII.GetBytes(nn);
            string s = null;
            foreach (byte b in keyBytes)
            {
                //Console.WriteLine(b);
                s += b.ToString() + ", ";
            }

            Console.WriteLine(s);
           
            Console.ReadKey();

            Environment.Exit(0);
            //*/
            Console.OutputEncoding = Encoding.UTF8;
            StringBuilder gen = new StringBuilder();
            gen.Append("\"ALL\"");
            var f = File.ReadAllLines("projects.xml");
            Dictionary<string, string> map = new Dictionary<string, string>();
            foreach (var l in f)
            {
                var nl = l.Trim();
                if (nl.StartsWith("<project code=") && nl.Contains("name="))
                {
                    var n = nl.Split('=');
                    var code = n[1].Replace("\"", null).Replace("name", null).Trim().ToUpper();
                    var name = n[2].Replace("\"", null).Replace("segment", null)
                        .Replace(">", null).Replace("-", "/")
                        .Replace(" Chine)", " (Chine)")
                        .Replace(" / ", "/").Replace("�", "ô")
                        .Trim();
                    map[code] = name;
                }
            }
            var orderedDictionary = map.OrderBy(pairs => pairs.Value);
            foreach (var l in orderedDictionary)
            {
                gen.Append(", \"" + l.Key + " - " + l.Value + "\"");
                Console.WriteLine(l.Key + " - " + l.Value);
                //Console.ReadKey();
                //Environment.Exit(0);
            }
            File.WriteAllText("vehicles.txt", gen.ToString());
        }
    }
}
```

**Reply by Furtif** [reply]
logically no old code will read this correctly as it has been remodeled to be read differently, I have read some that say "I click on the car and it doesn't!!" normally the condition has changed, there are more cars too, there are 4k and more ecus in data base... nothing has to do with the init version of this... if you don't know how to use the repo in real time with

git pull
and/ or ddt4all.bat or *.sh available in clone repo with proper python version is bad!

very easy thing with windows install ...

I also have the macOs version for beginners...

https://drive.google.com/file/d/1y46RNGut0HuXhUXsYMMo69qQ0-t7Nv9L/view?usp=drive_link

**Reply by Furtif** [reply]
![image](https://github.com/cedricp/ddt4all/assets/11375035/78dbfab3-dc82-4678-aa21-1a12da5c5dc6)

**Reply by Furtif** [reply]
well I'm not going to give a free class... I've already said too much, and everything is there, easy, effective, of course.

stay safe

**Reply by matzel82** [reply]
> the problem with a win xp, vista, 7, 8 system is the https or root certificates.. hence the site is http: and not https: just to be compatible.
> 
> 
> 
> ![image](https://github.com/cedricp/ddt4all/assets/11375035/5b348830-518b-4d7f-9706-9c66d54b3951)
> 
> 
> 
> then I don't know .... how do you come on github ??? this is https, i assume its not with your win 7
> 
> 
> 
> the party that has used legacyupdate otherwise does not have access to 95% of the sites on (443) https

With win11 pc at home. At work I have win7

**Reply by Furtif** [reply]
![image](https://github.com/cedricp/ddt4all/assets/11375035/c315657e-81b3-4212-9abc-419b9961b365)

I even tried putting this on an ipad/iphone ... but the problem is really PyQT5 I also did the same for pyren @shrlnm  ... but better to switch from tk to kivy for that... alias @shrlnm  I have an old demo in py 2. x kivy on your part MOD_ddt* I can eventually give you access, maybe I'll give you ideas to change Pyren tk to kivy ... by the way I mentioned this in gitlab some time ago  [Tkinter black screen on Mac OS](https://gitlab.com/py_ren/pyren/-/issues/50#note_1161545379)

**Reply by Furtif** [reply]
> > the problem with a win xp, vista, 7, 8 system is the https or root certificates.. hence the site is http: and not https: just to be compatible.
> > ![image](https://user-images.githubusercontent.com/11375035/245247487-5b348830-518b-4d7f-9706-9c66d54b3951.png)
> > then I don't know .... how do you come on github ??? this is https, i assume its not with your win 7
> > the party that has used legacyupdate otherwise does not have access to 95% of the sites on (443) https
> 
> With win11 pc at home. At work I have win7
https://github.com/cedricp/ddt4all/discussions/845#discussioncomment-6155962

**Reply by Furtif** [reply]
don't get bored with all this chinese lol ... before closing tell me if you got it, if that's the case you can close it.

**Reply by matzel82** [reply]
It doesn't work with this machine. I will use first time a second, and than I have time to make it new. Python started and closed at the same time. The DLL I have copied to the system 32 folder. System is Up to date ...I think.

It doesn't load new update.

**Reply by Furtif** [reply]
> It doesn't work with this machine. I will use first time a second, and than I have time to make it new. Python started and closed at the same time. The DLL I have copied to the system 32 folder. System is Up to date ...I think.
> 
> It doesn't load new update.

I know and understand on virgin machines in dev (ie basic use) 80% of real deps are missing, microsoft has always been like this, it's not new, I assure you, I don't know your machine, do you have all the updates before the win7 support closes? to know it turn on the machine connect it to the net and do as above https://github.com/cedricp/ddt4all/discussions/845#discussioncomment-6155962 or try to install deps manually .. or leave it as py 2.x 1° release or 2° and uses ecu.zip in . 

but. win7 works as py3.9-32 bits i have a HP2600 (xp key licence pro orig) i have up to win7 cpu is arch 32bits and runs win7 ultimate + a lot shits but google chrome KO normal, ie KO normal, but Opera works for all sites or more or less depends also all rest...

**Reply by Furtif** [reply]
you marked this as answered, that's correct, but if you have a problem for me it's not correct, see that well, and don't get mad at my "half chinese" phrases as I have it working on win7 32bits I don't see why not you can't to have.

**Reply by Furtif** [reply]
the worst case takes the 32 bit version and installs https://www.python.org/downloads/windows/ 
maybe this will install the dep you want ... in the case we are talking about is 3.9 - 32 bits

**Reply by Furtif** [reply]
[Python 3.9.17 - June 6, 2023](https://www.python.org/downloads/release/python-3917/)
Note that Python 3.9.17 cannot be used on Windows 7 or earlier.

**Reply by Furtif** [reply]
in win install is 3.9.0 the first 3.9.x  but read all is == to last 3.8.xx or https://www.python.org/downloads/release/python-3811/

**Reply by Furtif** [reply]
![image](https://github.com/cedricp/ddt4all/assets/11375035/63ffe21d-a162-4b66-9b51-cf3eebd8d79f)
3.9.0 32 bits is inclued in install but ask a dep

**Reply by matzel82** [reply]
> you marked this as answered, that's correct, but if you have a problem for me it's not correct, see that well, and don't get mad at my "half chinese" phrases as I have it working on win7 32bits I don't see why not you can't to have.

I will try my best to understand. I have learned PC Thinks myself at 1999 ☺️

There a thinks who are very strange...like this. I have now use the older release with py2.7 and I copied the Ecu.zip to folder. These aren't found at the list 🙈

Tomorrow I will test the latest release at the other PC. I will see what happens.

**Reply by Furtif** [reply]
> It doesn't work with this machine. I will use first time a second, and than I have time to make it new. Python started and closed at the same time. The DLL I have copied to the system 32 folder. System is Up to date ...I think.
> 
> It doesn't load new update.

I don't think it should be put in systme32 ... is this part of microsoft's python net framework or is it part of who installed visual studio 2017 at the time, as it is now with visual studio 2022 etc. is it part of framework 4.6, 4.8, netcore3. 11, etc, we say that a person who installs visual studio (utility development by microsoft) depending on the chosen packs "this" dependency is done, now win7 autonome was really the 3.8.x and not the 3.9 ... but the 3.9 was and it is usable in win 7 i mean 3.9.0 not following

**Reply by Furtif** [reply]
[Python 3.9.0 - Oct. 5, 2020](https://www.python.org/downloads/release/python-390/)
Note that Python 3.9.0 cannot be used on Windows 7 or earlier. by python site but by microsoft yep

**Reply by Furtif** [reply]
automome mode is : 
[Python 3.8.6 - Sept. 24, 2020](https://www.python.org/downloads/release/python-386/)
Note that Python 3.8.6 cannot be used on Windows XP or earlier. 

32 bits for you

**Reply by Furtif** [reply]
go try make it... but...... recent code maybe as bug .. but go try same

**Reply by Furtif** [reply]
> I will try my best to understand. I have learned PC Thinks myself at 1999 ☺️
> 
> There a thinks who are very strange...like this. I have now use the older release with py2.7 and I copied the Ecu.zip to folder. These aren't found at the list 🙈
> 
> Tomorrow I will test the latest release at the other PC. I will see what happens.
normal... 
we are a little m 2023 almost halfway through

**Reply by Furtif** [reply]
back to: delete all and forget it, I'll do a win install with py 3.8.6, with almost all the last changes in code.

if ok autonome py codes win7 without visual studios is 3.8.6

**Reply by alxlve** [reply]
@Furtif Hello. Could I get access to your private repository? Thank you.

**Reply by atismert222** [reply]
please new database?

**Reply by mrkmhgh** [reply]
Hi the link for ecu.zip has expired. Any chance of a re-up?
thanks

**Reply by AndyIreland** [reply]
> Hi the link for ecu.zip has expired. Any chance of a re-up? thanks

hi, see if this works:
https://drive.google.com/file/d/1Zy4OVUcOawkGlI_Tcj18Ztj0kKQex8eA/view?usp=sharing

**Reply by Renotech1** [reply]
https://www.swisstransfer.com/d/be8adebc-b54c-4744-87cc-9c24939d56de

**Reply by igogo2** [reply]
> https://www.swisstransfer.com/d/be8adebc-b54c-4744-87cc-9c24939d56de

Link is expired. Please republish.

**Reply by SDDZ213** [reply]
Hello please send me the latest version of the database ( ecu.zip ) Thanks

**Comment by Furtif** [comment]
New release is now available https://github.com/cedricp/ddt4all/releases/tag/v2.1

**Comment by Furtif** [comment]
well try ... https://github.com/cedricp/ddt4all/releases/tag/v2.1.1 but it need paste bd.zip in

**Comment by Furtif** [comment]
if ok close now.

**Reply by matzel82** [reply]
On a other win7 it's work perfectly. 32 bit and py 3.8

I love it 🥰

**Reply by Furtif** [reply]
> On a other win7 it's work perfectly. 32 bit and py 3.8
> 
> I love it 🥰

Cool 👌

**Comment by KarelSvo** [comment]
> 
> Tomorrow I will test the latest release at the other PC. I will see what happens.


Which vehicle do you want to connect to ?  I only build python 2.7 versions covering all Windows versions from XP onwards.
And I only insert the vehicles and ecus I need.

I only use the latest version of an ECU file.
It is important to know that the ECU database was not written for the end user. The list of software versions for the automatic identification of the ecus is mostly incomplete or has spelling mistakes. So then no match is possible during automatic scanning.

**Reply by Furtif** [reply]
> > Tomorrow I will test the latest release at the other PC. I will see what happens.
> 
> Which vehicle do you want to connect to ? I only build python 2.7 versions covering all Windows versions from XP onwards. And I only insert the vehicles and ecus I need.
> 
> I only use the latest version of an ECU file. It is important to know that the ECU database was not written for the end user. The list of software versions for the automatic identification of the ecus is mostly incomplete or has spelling mistakes. So then no match is possible during automatic scanning.

Yep ddt2000 stuff also uses only ecu setted in ecu/eculist.xml 

the thing that seems bad to me is that ddt4all uses address/address.json instead of using vehicle/XXX/addressing.xml , but the json generated with the generic has all addresses theoretically.

**Comment by eldimoni1** [comment]
Ante todo perdón por la pregunta me he leído todo el tocho, y aún asi no lo he entendido bien, 
bajándome la ecu que ponéis serviría para mejorar el Dd4all que tengo? Con fecha del 2021?
Gracias.un saludo

**Reply by Furtif** [reply]
> Ante todo perdón por la pregunta me he leído todo el tocho, y aún asi no lo he entendido bien, bajándome la ecu que ponéis serviría para mejorar el Dd4all que tengo? Con fecha del 2021? Gracias.un saludo

normalmente sí, ahora si tienes una PC moderna puedes usar la última, recuperar el nuevo archivo [projects.json](https://github.com/cedricp/ddt4all/blob/master/ddt4all_data/projects.json) (cambiar el existente) y usar el mismo

**Reply by eldimoni1** [reply]
> > Ante todo perdón por la pregunta me he leído todo el tocho, y aún asi no lo he entendido bien, bajándome la ecu que ponéis serviría para mejorar el Dd4all que tengo? Con fecha del 2021? Gracias.un saludo
> 
> normalmente sí, ahora si tienes una PC moderna puedes usar la última, recuperar el nuevo archivo [projects.json](https://github.com/cedricp/ddt4all/blob/master/ddt4all_data/projects.json) (cambiar el existente) y usar el mismo

Hola ya lo he bajado he instalado en Windows 10, y si me salen, más variantes dentro de la misma ecu, , lo que no sé, si valdría para el Clío 5, o son para otros modelos, 

Gracias

**Reply by Furtif** [reply]
> > > Ante todo perdón por la pregunta me he leído todo el tocho, y aún asi no lo he entendido bien, bajándome la ecu que ponéis serviría para mejorar el Dd4all que tengo? Con fecha del 2021? Gracias.un saludo
> > 
> > 
> > normalmente sí, ahora si tienes una PC moderna puedes usar la última, recuperar el nuevo archivo [projects.json](https://github.com/cedricp/ddt4all/blob/master/ddt4all_data/projects.json) (cambiar el existente) y usar el mismo
> 
> Hola ya lo he bajado he instalado en Windows 10, y si me salen, más variantes dentro de la misma ecu, , lo que no sé, si valdría para el Clío 5, o son para otros modelos,
> 
> Gracias

Vale, reemplace proyectos.json con lo que ahora está en los repositorios, es más detallado y tiene buenas direcciones.
Lo original está bien, pero hay gente que le gusta tener el código primero, así que les digo que lo hagan así.

hasta pronto

**Comment by ernestzz** [comment]
is there a new version for 2025?

**Comment by brkicdanilo** [comment]
Can You suggest me where to find ecu in xml format for DDT4All windows version?

**Reply by AndyIreland** [reply]
I think its in .json format for DDT4ALL - if you download pyren183 from here 

https://www.drive2.ru/b/501493851572142178/

the .xml are in the ecu folder though there once you have installed it:
Program Files (x86)\PyREN\ecus

**Reply by Furtif** [reply]
@AndyIreland  ddt4all works also as xml files versions.... need only delete ecu.zip and put into root clone repo the content of c:/DDT2000Data

**Comment by brkicdanilo** [comment]
Thanks,
Regards


čet, 3. jul 2025. 11:56 --=FurtiF™=-- ***@***.***> je
napisao/la:

**Comment by atismert222** [comment]
Yes, please.

Android için Outlook<https://aka.ms/AAb9ysg> edinin
________________________________

**Comment by cristyanul** [comment]
Do you have a working link?

**Comment by steph2monaco** [comment]
Bonjour à tous, je souhaite savoir si c'est possible d'avoir la dernière version de la base DDT2000 pour DDT4all svp? Je souhaiterai consulter pour un renault kadjar de 2016. Merci

**Reply by Renotech1** [reply]
Send private for link

**Reply by steph2monaco** [reply]
Sorry for my ignorance, but how do I send a private message?

**Reply by AndyIreland** [reply]
> Hello everyone, I'd like to know if it's possible to get the latest version of the DDT2000 database for DDT4all, please? I'd like to use it for a 2016 Renault Kadjar. Thank you.

Here is one to try, rename it to ecu.zip:
https://drive.google.com/file/d/1GBxqDIiG8hvZ-cKBUrcKq7l0b6EmYf2E/

If that dont work try this one:
https://drive.google.com/file/d/15YD_ghvfS37uuOqvlLJNr4ypZqOKLqVv/

Good luck

**Reply by steph2monaco** [reply]
Thanks for your help; I'll give it a go and let you know how it goes
